### DISCUSSION TITLE
possible to roll back to previous cask version?

### DISCUSSION THREAD

**Discussion by maichai** [Homebrew/discussions#2379] [Casks] [answered]
If accidently having installed a new version of a cask that is not working (e.g. because it only works with a macOS version not compatible with the machine), is it possible to roll back to the last installed cask version?

**Comment by SMillerDev** [comment, marked as answer]
Not within brew, but if your cask has a versioned download you can just download and install an older package.

**Comment by maichai** [comment]
yes the software has a versioned download, i figured that out, but I wanted to do it within brew. so no luck.
